.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -43,4 +43,3 @@ author(s) can easily look it up.-->
 ## For Elastic Internal Use Only
 - [ ] Considered corresponding documentation changes to [contribute separately](https://github.com/elastic/enterprise-search-pubs#contribute-docs-changes-for-product-changes)
 - [ ] New configuration settings added in this PR follow the [official guidelines](https://github.com/elastic/ent-search/blob/main/doc/enterprise-search-config.md)
-- [ ] [Manual test steps](https://github.com/elastic/connectors-ruby/blob/main/docs/INTERNAL.md#minimal-manual-tests) are successful
```

Comment:
Why is this going away? Not doing these sort of manual test steps is what recently caused a scramble at FF when Crawler needed to update its usage of the connectors-utility gem, but couldn't update because there'd been some breaking changes left undiscovered.

Reply:
`INTERNAL.md` is gone, and #minimal-manual-tests in `INTERNAL.md` is TBD.

I actually don't see other manual tests necessary besides
1. `make lint test`, which is covered by CI
2. https://github.com/elastic/connectors-ruby/blob/d95b7003caff6c34543ca64214ec4ca04a093426/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L17

>Not doing these sort of manual test steps is what recently caused a scramble at FF when Crawler needed to update its usage of the connectors-utility gem, but couldn't update because there'd been some breaking changes left undiscovered.

Oh, you meant to test the utility gem in ent-search. I can add it here, but I expect this will eventually be gone when crawler code is migrated to this repo completely.

Reply:
👍 as long as we have a checkbox somewhere that covers testing that the connectors changes don't break ent-search, I've got not problem. :) I agree that this is a short-ish term thing, and that we won't need to do this once Crawler moves, but until then, it would be great to find problems sooner than later.